engine/src/test/java/org/camunda/bpm/engine/test/api/mgmt/metrics/JobExecutorMetricsTest.java

```diff
@@ -147,23 +165,33 @@ public void testJobExecutionMetricReporting() {
     }
 
     // when
-    testRule.waitForJobExecutorToProcessAllJobs(5000);
+    // extend waiting time for CRDB since it takes longer to process all the jobs there
+    // see CAM-12239 for more details
+    long maxMillisToWait = testRule.isOptimisticLockingExceptionSuppressible()? 5000L : 30000L;
```

Comment:
This test is still flaky, even though the wait time was increased. As we discussed before, I would exclude it since we already have other tests that cover the JobExecution metrics. The test is referenced in [CAM-12461](https://jira.camunda.com/browse/CAM-12461).